package.json

```diff
@@ -49,7 +49,9 @@
     "@uirouter/angularjs": "^1.0.15",
     "angular": "1.6.10",
     "angular-translate": "^2.17.0",
-    "lodash": "~3.0.1",
     "ng-slide-down": "^0.4.2"
+  },
+  "dependencies": {
```

Comment:
In order to keep consistency across all package.json files you can run: `npx sort-package-json`.

Reply:
Done. IMO, it should be automated.